Source/GSXib5KeyedUnarchiver.m

```diff
@@ -324,6 +326,7 @@ + (void) initialize
                                            @"beginningViews", @"NSStackViewBeginningContainer",  // NSStackView
                                            @"middleViews", @"NSStackViewMiddleContainer",
                                            @"endViews", @"NSStackViewEndContainer",
+                                           @"primaryBackgroundColor", @"NSBackgroundColors",
```

Comment:
The XmlKeyMapTable was really for defining a direct key mapping for names that were not easily translated.

NSBackgroundColors was originally an array that included the primary color (primaryBackgroundColor), and if defined, the secondary color (secondaryBackgroundColor).  Therefore, it was mapped to a decoder function (decodeBackgroundColorsForElement) in XmlKeyToDecoderSelectorMap in order to process the necessary attributes.  Adding this definition will break that processing.  Has something changed with the Apple NIBs?

Reply:
No. Nothing has changed.  I noticed this breakage and reverted that change.